### ISSUE TITLE
Enhancement request: Climax status Bar

### ISSUE THREAD

**Issue by TheKnightofAura** [naelstrof/KoboldKare#167]
While I agree that a cum button is a little bit against the spirit of the game, I would love to have a small progress bar, ideally located just over the (press Z or X to adjust arousal) buttons that slowly fills as you near a climax, and then, when it is full and you nut, slowly drains as a timer measuring how long you're going to nut for.

This bar could also double as an indicator for how long a tie is going to last when knots are given that function.

**Comment by naelstrof** [comment]
Yeah there definitely needs to be more info on when you're about to climax, even while I'm developing I'm not sure how my character is feeling while I'm testing things.
